src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const listAll = document.querySelector('ul');
+const list = document.querySelectorAll('li');
+
+function sortList(arr) {
+  const listArr
+   = [...arr].sort((a, b) => b.dataset.salary
+     .slice(1).split(',').join('') - a.dataset.salary
```

Comment:
For converting strings to numbers you should create a separate function (as the task description says)